Public/Src/Pips/Dll/MutablePipState.cs

```diff
@@ -37,14 +37,17 @@ internal class MutablePipState
 
         internal readonly PipType PipType;
 
+        internal readonly int Priority;

```

Comment:
>Priority [](start = 30, length = 8)

Seems like this only applies to process pips. Should probably go in ProcessMutablePipState.

Reply:
Do you think it should still be accessed through piptable or be a public property of processmutablepipstate

---
In reply to: [280853322](https://github.com/Microsoft/BuildXL/pull/239#discussion_r280853322) [](ancestors = 280853322)